Update Readme.MD, example not compilable

We have updated to the neweset version and changed the Types accordingly. However compilation is not possible becasue 

`this.dashboard = [
       {cols: 2, rows: 1, y: 0, x: 0},
       {cols: 2, rows: 2, y: 0, x: 2}
     ];`

is failing with following error message
X [ERROR] TS2353: Object literal may only specify known properties, and 'cols' does not exist in type 'GridsterItem'. [plugin angular-compiler]

    dashboard.component.ts:124:13:



Please update Readme.MD to change type to GridsterItemConfig for dashboard variable.

Thanks

--- a/README.md
+++ b/README.md
@@ -45,15 +45,15 @@ import {Gridster, GridsterItem} from 'angular-gridster2';
 Initialize a simple dashboard:
 
 ```typescript
-   import { GridsterConfig, GridsterItem }  from 'angular-gridster2';
+   import { GridsterConfig, GridsterItemConfig }  from 'angular-gridster2';
    options: GridsterConfig;
-   dashboard: GridsterItem[];
+   dashboard: GridsterItemConfig[];
 
-   static itemChange(item, itemComponent) {
+   static itemChange(item: GridsterItemConfig, itemComponent) {
      console.info('itemChanged', item, itemComponent);
    }
 
-   static itemResize(item, itemComponent) {
+   static itemResize(item: GridsterItemConfig, itemComponent) {
      console.info('itemResized', item, itemComponent);
    }
 
@@ -73,12 +73,12 @@ Initialize a simple dashboard:
      this.options = Object.assign({}, this.options);
    }
 
-   removeItem(item) {
+   removeItem(item: GridsterItemConfig) {
      this.dashboard.splice(this.dashboard.indexOf(item), 1);
    }
 
    addItem() {
-     this.dashboard.push({});
+     this.dashboard.push({cols: 1, rows: 1, y: 0, x: 0});
    }
 ```
 

